include/QuartzCore/CARenderer.h

```diff
@@ -15,12 +15,12 @@
 //******************************************************************************
 #pragma cone
 
+#import <CoreVideo/CVTime.h>
```

Comment:
>CoreVideo [](start = 9, length = 9)

rather than introducing this dependency, we should simply change the forward declaration below to be struct CVTimeStamp.